Automattic/wp-calypso
# Added UI controls for Edge Cache Defensive Mode

Fixes https://github.com/Automattic/dotcom-forge/issues/9077

## Proposed Changes

Introducing UI controls for Edge Cache Defensive Mode on the Server Settings page (`/hosting-config/:site_slug`)

| Edge cache disabled | Disabled | Enabled |
| - | - | - |
| ![edge-cache](https://github.com/user-attachments/assets/63b9e435-fd03-42fd-8ca9-6dbe616bebfa) | ![disabled](https://github.com/user-attachments/assets/74e7767c-c8d4-4190-b795-39cb3efd5443) | ![enabled](https://github.com/user-attachments/assets/94410912-14f0-48e9-a252-ac0a763711c2) |

## Why are these changes being made?


See pcYYuD-1Sy-p2.

## Testing Instructions



1. Have an Atomic site
2. Navigate to `/hosting-config/:site_slug`
3. Disable edge cache
4. Ensure that the `Defensive mode` card contains the following text:

> Defensive mode requires [global edge cache](https://developer.wordpress.com/docs/site-performance/global-edge-cache/) to be enabled.

5. Enable edge cache
6. Enable edge cache defensive mode
7. Navigate to your website
8. Ensure that you see a brief `Checking your browser` page
9. Disable edge cache
10. Navigate to your website in a different browser
11. Ensure that you **do not** see a brief `Checking your browser` page
